plugin/builtin/replicacounttransformer/ReplicaCountTransformer_test.go

```diff
@@ -148,9 +149,8 @@ spec:
 }
 
 func TestMatchesCurrentID(t *testing.T) {
-	th := kusttest_test.MakeEnhancedHarness(t).
-		PrepBuiltin("PrefixSuffixTransformer").
-		PrepBuiltin("ReplicaCountTransformer")
```

Comment:
What does `PrepBuiltin` do? I find it odd that you had to remove line 153 here - `PrepBuiltin("ReplicaCountTransformer` - since we aren't touching the ReplicaCountTransformer.

Reply:
`PrepBuiltin` generates a `.so` for the given transformer which is not necessary when using `types.BploUseStaticallyLinked` as the loading option. We use `types.BploUseStaticallyLinked` because of the `PrefixSuffixTransformer` used in this test but because of this, calling `PrepBuiltin` for the `ReplicaCountTransformer` is not necessary either anymore.

@KnVerey, what do you think? I'm not too familiar with the `PrepBuiltin` (only the parts I worked with and debugged) and I might be talking gibberisch 😄

Reply:
@Serializator that sounds right to me. Agreed that we shouldn't need either in this test now that it uses the statically linked copy.